File: src/main/java/kr/co/bookand/backend/BookandApplication.java
package kr.co.bookand.backend;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.context.annotation.Bean;
import org.springframework.scheduling.annotation.EnableScheduling;
import org.springframework.web.client.RestTemplate;

@EnableScheduling
@SpringBootApplication
public class BookandApplication {

	@Bean
	public RestTemplate restTemplate() {
		return new RestTemplate();
	}

	public static void main(String[] args) {
		SpringApplication.run(BookandApplication.class, args);
	}

}

File: src/main/kotlin/kr/co/bookand/BookandApplication.kt
package kr.co.bookand

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.context.annotation.Bean
import org.springframework.data.jpa.repository.config.EnableJpaAuditing
import org.springframework.scheduling.annotation.EnableScheduling
import org.springframework.web.client.RestTemplate


@EnableScheduling
@EnableJpaAuditing
@SpringBootApplication
class BookandApplication

fun main(args: Array<String>) {
    runApplication<BookandApplication>(*args)
}

